### PR TITLE
Initialize deployment

### PR DESCRIPTION
**Pull request by daystram**

### CODE REVIEW

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+      - name: Set Development Version
+        run: echo "RELEASE_VERSION=v0.0.0-development" >> $GITHUB_ENV
+      - name: Provide Env File
+        run: |
```
Comment (nicolauscg) [inline .github/workflows/nightly.yml:31]: is the char '|' intentional here? if so whats its purpose?

File: .github/workflows/nightly.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          echo "${{ secrets.FE_ENV_FILE }}" > ${{ env.APPLICATION }}-${{ matrix.service }}/.env.production
+          echo "REACT_APP_VERSION=${{ env.RELEASE_VERSION }}" >> ${{ env.APPLICATION }}-${{ matrix.service }}/.env.production
+      - name: Build and Push Image
```
Comment (nicolauscg) [inline .github/workflows/nightly.yml:34]: the build step just runs npm and go commands to build the app without Docker.

with this build image step, for frontend, do we just need a Dockerfile under fableous-fe/ for it to work?

File: .github/workflows/nightly.yml (hunk 1 above)
Comment (daystram) [reply .github/workflows/nightly.yml:31]: Multiline string in YAML

File: .github/workflows/nightly.yml (hunk 2 above)
Comment (daystram) [reply .github/workflows/nightly.yml:34]: Yea you need Dockerfile. Nightly workflow pushed dev image to the hub.

**Comment by daystram** [comment]
Waiting for frontend init

File: README.md [hunk 3]
```diff
@@ -1 +1,107 @@
...
+$ air
+```
+
+### fableous-fe
+
+To begin developing, simply enter the sub-directory and run the development server:
+
+```shell
+$ cd fableous-fe
+$ npm install
```
Comment (nicolauscg) [inline README.md:34]: in init frontend branch, frontend details in readme should be rechecked (I'll do it before PR)

File: .github/workflows/nightly.yml (hunk 2 above)
Comment (nicolauscg) [reply .github/workflows/nightly.yml:34]: yep just add Dockerfile

**Review by nicolauscg — APPROVED** [review]
changes looks good aside from few qs, already resolved though

File: README.md (hunk 3 above)
Comment (nicolauscg) [reply README.md:34]: I think the only change needed is to use yarn instead of npm here

**Review by nicolauscg — APPROVED** [review]
changes looks fine

**Pull request merged**